local-cli/generator-windows/templates/index.windows.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
```

Comment:
Not saying we have to do it now, but we should also import something from react-native-windows in this sample so people see how it's done.